qokit/utils.py

```diff
@@ -13,10 +13,12 @@
 from pytket.extensions.quantinuum import QuantinuumBackend
 from pytket.extensions.qiskit import qiskit_to_tk
 from importlib_resources import files
+from qiskit.providers.basic_provider import BasicProvider
 
 from pytket.passes import (
     SequencePass,
-    auto_squash_pass,
+    AutoSquash,
+    # auto_squash_pass,
```

Comment:
Can you please remove commented out code?

Reply:
Done!